src/shared/components/post/post-listings.tsx

```diff
@@ -69,13 +82,108 @@ interface PostListingsProps {
   onHidePost(form: HidePost): void;
   onPersonNote(form: NotePerson): void;
   onScrollIntoCommentsClick(): void;
+  onNextPage?(): void;
+  onPrevPage?(): void;
+  onFirstPage?(): void;
 }
 
-export class PostListings extends Component<PostListingsProps, any> {
+@keyboardShortcutsMixin
+export class PostListings
+  extends Component<PostListingsProps, KeyboardNavigationState>
```

Comment:
Rename this to PostListingsState or somethin. Keyboardnavigationstate is feature, not this componens.